src/project-2.js

```diff
@@ -3,6 +3,13 @@
 const getBiggest = (x, y) => {
   // x and y are integers.  Return the larger integer
   // if they are the same return either one
+  if (x === y) {
+    return x;
```

Comment:
There is a `>=` operator so you can combine those two `if` statements like this:

```
if (x >= y) return x;
return y;
```
OR you can use a ternary operator
```
const getBiggest = (x, y) => x >= y ? x : y;
```